catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -306,9 +306,8 @@
     <bean id="deprecatedProductCache" class="ddf.catalog.cache.impl.ResourceCacheImpl"
           init-method="setupCache"
           destroy-method="teardownCache">
-        <property name="productCacheDirectory" value=""/>
-        <property name="cacheDirMaxSizeMegabytes" value="10240"/>
```

Comment:
Actually it looks like it's still there, I think I actually lost the `cacheDirMaxSizeMegabytes` property